app/utils/base_client2.js

```diff
@@ -74,8 +74,9 @@ const _request = function request (methodName, url, args, callback, subSegment)
   const namespace = getNamespace(clsXrayConfig.nameSpaceName)
   const clsSegment = namespace ? namespace.get(clsXrayConfig.segmentKeyName) : null
 
+  const proxiedUrl = addProxy(url)
   const requestOptions = {
-    uri: addProxy(url),
+    uri: proxiedUrl.protocol + '//' + proxiedUrl.hostname + ':' + proxiedUrl.port + proxiedUrl.pathname,
```

Comment:
there might be a better way to do this but this is breaking in production at the moment and am keen to get a quick fix out.